project.py

```diff
@@ -81,6 +86,21 @@ def find_board(self, image):
 
         return our_cnt
 
+    def get_seed(self, image):
+        if self.seed is not None:
+            rgb = image[self.seed[1], self.seed[0]]
+            if rgb[1] > rgb[0] and rgb[1] > rgb[2]:
```

Comment:
@Foxiy This is a nice example of ghetto green filter but I believe a comment here would not hurt as it is not obvious right away what does this thing do.